pkgs/servers/clickhouse/default.nix

```diff
@@ -29,13 +29,6 @@ stdenv.mkDerivation rec {
     "-DUSE_INTERNAL_SSL_LIBRARY=False"
   ];
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/yandex/ClickHouse/commit/afbcdf2f00a04e747c5279414cf4691f29bb5cc2.patch";
-      sha256 = "17y891q0dp179w3jv32h74pbfwyzgnz4dxxwv73vzdwvys4i8c8z";
-    })
-  ];
-
   postPatch = ''
     patchShebangs copy_headers.sh
```

Comment:
This line has to be updated to `patchShebangs dbms/programs/clang/copy_headers.sh`.

Reply:
done